test/tree-toggle.html

```diff
@@ -158,6 +169,32 @@
       });
 
     });
+
+    describe('async recalculateWidth columns', () => {
+      let grid;
+    
+      beforeEach(() => {
+        grid = fixture('aysnc-tree-grid');
+      });
+    
+      it('should recalculate width columns when children nodes rendered', () => {
```

Comment:
This test doesn't relate to tree-toggle so it shouldn't be in this file. Relocate to `column-auto-width`. Fix the description typos "should recalculate column widths when child items loaded"

Reply:
moved it. thank you for correcting.